src/routes/+page.svelte

```diff
@@ -132,6 +139,7 @@
 		<Field name="danger-zone">
 			<svelte:fragment slot="label">Danger zone</svelte:fragment>
 			<Button variant="outline" on:click={deleteSessions}>Delete all sessions</Button>
+			<Button variant="outline" on:click={deleteKnowledge}>Delete all knowledge</Button>
```

Comment:
Button's will end up looking something like this:

```suggestion
<Button variant="outline" on:click={()=> deleteStorage(StorageKey.HOLLAMA_KNOWLEDGE}>Delete all knowledge</Button>
```